draft-ietf-quic-qpack.md

```diff
@@ -715,6 +714,132 @@ None.
 
 --- back
 
+# Static Table
+
+| Index | Name                             | Value                                                                                                  |
+| ----- | -------------------------------- | ------------------------------------------------------------------------------------------------------ |
+| 0     | :path                            | /                                                                                                      |
+| 1     | referer                          |                                                                                                        |
+| 2     | content-length                   | 0                                                                                                      |
+| 3     | date                             |                                                                                                        |
+| 4     | last-modified                    |                                                                                                        |
+| 5     | :authority                       |                                                                                                        |
+| 6     | expires                          |                                                                                                        |
```

Comment:
Considering `Expires` is HTTP/1.0, maybe don't include it to encourage people to start using `Cache-Control: max-age`?